draft-ietf-quic-http.md

```diff
@@ -249,27 +249,16 @@ the HTTP framing layer. A QUIC receiver buffers and orders received STREAM
 frames, exposing the data contained within as a reliable byte stream to the
 application.
 
-QUIC reserves the first client-initiated, bidirectional stream (Stream 0) for
-cryptographic operations. HTTP over QUIC reserves the first unidirectional
-stream sent by either peer (Streams 2 and 3) for sending and receiving HTTP
-control frames.  This pair of unidirectional streams is analogous to HTTP/2's
-Stream 0.  HTTP over QUIC also reserves the second and third unidirectional
-streams for each peer's QPACK encoder and decoder.  The client's QPACK encoder
-uses stream 6 and decoder uses stream 10.  The server's encoder and decoder use
-streams 7 and 11, respectively.  The data sent on these streams is critical to
-the HTTP connection. If any control stream is closed for any reason, this
-MUST be treated as a connection error of type QUIC_CLOSED_CRITICAL_STREAM.
-
 When HTTP headers and data are sent over QUIC, the QUIC layer handles most of
 the stream management.
 
-An HTTP request/response consumes a single client-initiated, bidirectional
-stream.  A bidirectional stream ensures that the response can be readily
-correlated with the request. This means that the client's first request occurs
-on QUIC stream 4, with subsequent requests on stream 8, 12, and so on.
-
-Server push uses server-initiated, unidirectional streams.  Thus, the server's
-first push consumes stream 7 and subsequent pushes use stream 11, 15, and so on.
+QUIC reserves the first client-initiated, bidirectional stream (Stream 0) for
+cryptographic operations. All remaining client-initiated bidirectional streams
+are used for requests and responses.  A bidirectional stream ensures that the
+response can be readily correlated with the request. This means that the
+client's first request occurs on QUIC stream 4, with subsequent requests on
+stream 8, 12, and so on. HTTP/QUIC does not use server-initiated bidirectional
```

Comment:
I don't really like the comma in that construction, since it's describing which flavor of Ydirectional stream rather than two independent descriptors of the stream.  So I fixed it by removing the other commas.  😉

"HTTP/QUIC" is used 60 times in this document, "HTTP over QUIC" is used 5.  I think of them equivalently to "HTTP/2" and "HTTP version 2" in RFC7540.  I'll do a pass separately to see if it's appropriate to trim any other "over"s.